README.md

```diff
@@ -1,2 +1,76 @@
-# intergalactic-most-wanted-list
-A database for malware checksums for use with the WallE malware cleanup script
+# Intergalactic Most Wanted List
+A database for malware checksums for use with the [WallE](https://github.com/usegalaxy-eu/WallE) malware cleanup script.
+
+This database in form of a yaml file stores malware checksums and its metadata
+schema:
+~~~yaml
+class:
+  program:
+    version:
+      severity: [high, medium, low]
+      description: "optional info"
+      checksums:
+        crc32: <checksum crc32, gzip algorithm, integer representation>
+        sha1: <checksum sha1, hex representation>
+~~~
+Contributions are always welcome. Here is a more detailed explanation how the schema works.
+### Class
+What purpose does it serve, e.g. mining, (port-)scanner, multi-purpose...  
+
+**Not** a `class` would be `nmap`, it is a specific tool for scanning and would be a `program` under the `scanning` class.
+
+### Program
+I did not come up with a better title for now. (Refused to call it 'tool')  
+Feel free to rename this.  
+
+A `program` is every single unit of malware. In most cases it would be an executable file. But it can be a script, a binary, a package or a zip archive.  
+Think of everything that you would version.
+
+### Version
+
+The version of the program. (Not much more to say, I guess?)
```

Comment:
I it is important to specify one, because the hash can change and that would indicate a new "version". But I can add that people can give it a version themselves, if there is no version